Keras_Tensorflow/01_DevelopModelDriver.ipynb

```diff
@@ -212,8 +206,8 @@
       "DEBUG:PIL.PngImagePlugin:iCCP profile name b'ICC Profile'\n",
       "DEBUG:PIL.PngImagePlugin:Compression method 0\n",
       "DEBUG:PIL.PngImagePlugin:STREAM b'IDAT' 345 65536\n",
-      "INFO:model_driver:Predictions: [{'image': [('n02127052', 'lynx', 0.98164833), ('n02128385', 'leopard', 0.0077441484), ('n02123159', 'tiger_cat', 0.0036861449)]}]\n",
-      "INFO:model_driver:Predictions took 113.41 ms\n"
+      "INFO:model_driver:Predictions: [{'image': [('n02127052', 'lynx', 0.9816487), ('n02128385', 'leopard', 0.007744099), ('n02123159', 'tiger_cat', 0.0036861112)]}]\n",
+      "INFO:model_driver:Predictions took 3110.29 ms\n"
```

Comment:
@msalvaris All the timings are super slow, Mario used NC6 DLVM, would that be why?

Reply:
The first run is always super slow. Run it twice